backend/src/Entity/Fishbowl.php

```diff
@@ -40,7 +40,8 @@
  *     normalizationContext={"groups"={"fishbowl:read"}},
  *     denormalizationContext={"groups"={"fishbowl:write"}},
  *     collectionOperations={
- *         "post"={"security"="is_granted('ROLE_USER')"}
+ *         "get",
```

Comment:
You should take a look at this: https://api-platform.com/docs/core/operations/#enabling-and-disabling-operations